app/views/users/invitations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2><%= t "devise.invitations.new.header" %></h2>
+
+<%= form_for(resource, as: resource_name, url: invitation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="clearfix">
+    <%= f.label :first_name %>
+    <%= f.text_field :first_name, autofocus: true, required: true, autocomplete: :first_name %>
+  </div>
+
+  <div class="clearfix">
+    <%= f.label :middle_name %>
+    <%= f.text_field :middle_name, autofocus: true, autocomplete: :middle_name %>
+  </div>
+
+  <div class="clearfix">
+    <%= f.label :last_name%>
+    <%= f.text_field :last_name, autofocus: true, required: true, autocomplete: :last_name %>
+  </div>
+
+  <div class="clearfix">
```

Comment:
Does it make difference if we remove clearfix class?

Reply:
creates no difference.